### DISCUSSION TITLE
Should It Be Possible To Associate Multiple Items To One Asset ID?

### DISCUSSION THREAD

**Discussion by AstroPhotoJay** [sysadminsmedia/homebox#57] [General]
I've been playing around with HB as a tool to help keep track of my families personal possessions while moving house and I have discovered it is possible to associate multiple items to a single asset ID.

(To do this, simply edit the asset ID of an existing item to match that of another existing item)

I would suspect this would be undesirable for most use cases, but at the same time it could be a handy work around for some edge cases [such as this](https://github.com/sysadminsmedia/homebox/discussions/56).

So instead of opening an issue, I figured a discussion might be worthwhile.

**Comment by colbach** [comment]
I also just tried to figure out whether this to be a feature or bug. In my opinion the optimal behavior here would be a popup saying "This Asset ID is already in use, do you want to use it anyway?". I can see why someone might want to do this, but lets face that in 99% of the cases it will just happen out of mistake.

(Still one of the biggest issues for me right now when it comes to Asset ID is that I can't even search for an Asset ID. How does this make any sense?)

**Reply by tonyaellie** [reply]
![image](https://github.com/user-attachments/assets/518b30b0-8e32-4ef8-afdf-c01996b6f72a)

**Comment by tonyaellie** [comment]
Duplicating an item will also give the duplicate the same id though the duplicate having the same id is a bug.
